conference/ical.py

```diff
@@ -98,7 +98,7 @@ def __init__(self, name, *args, **kwargs):
 
     def encode(self):
         yield content('BEGIN', self.name)
-        for name, prop in self.items():
+        for name, prop in list(self.items()):
```

Comment:
Unnecessary `list`.

Reply:
For future reference: find . -type f -name \*.py -exec gawk -i inplace '{print gensub(/for (.+) in list\((.+)\):/, "for \\1 in \\2:", "g");}' {} \;